### PR TITLE
Remove installed phars on pkg uninstall

### PR DESCRIPTION
**Pull request by sebastianfeldmann**

### CODE REVIEW

File: src/ConfiguredMediator.php [hunk 1]
```diff
@@ -30,4 +39,22 @@ private function createInstallerFromConfig(Config $config, PackageEvent $event):
...
+
+        /** @var \PharIo\ComposerDistributor\File $phar */
+        foreach ($config->phars() as $phar) {
+            $pharLocation = $binDir . DIRECTORY_SEPARATOR . $phar->pharName();
+            if (is_file($pharLocation)) {
+                $this->io->write(sprintf(
+                    'remove phar \'%1$s\'',
+                    $phar->pharName()
+                ));
+                unlink($pharLocation);
```
Comment (theseer) [inline src/ConfiguredMediator.php:56]: What happens if we don't have write permissions here?

**Review by heiglandreas — APPROVED** [review]
:ship: :it:

**Review by theseer — CHANGES_REQUESTED** [review]
Only one question ;)

File: src/ConfiguredMediator.php (hunk 1 above)
Comment (sebastianfeldmann) [reply src/ConfiguredMediator.php:56]: good question, I will address this in another pull request ;)

Comment (heiglandreas) [reply src/ConfiguredMediator.php:56]: We can't unlink. But we were not able to link something there in the first place.... 

Much more interesting is the question whether that value could be modified to remove something "important". But that would mean that the XML-Config-file would need to be tampered with which should only be possible for the creator of the plugin... So: No. Doesn't look like an attack-vector to me....

Comment (theseer) [reply src/ConfiguredMediator.php:56]: Nobody said anything about attack vector. Just things that can happen: Vou could have run install things as root or any other user, changed ownership or permissions between install and remove, could have a read-only mounts within a container and try scripting remove, ....

Comment (heiglandreas) [reply src/ConfiguredMediator.php:56]: Oh that wasn't related. the attack vector just came to my mind when I thought about removing files... 😁

**Pull request merged**